nix/deps/gradle/proj.list

```diff
@@ -1,6 +1,7 @@
 app
 react-native-async-storage_async-storage
 react-native-background-timer
+react-native-biometrics
```

Comment:
hmm, I'm surprised to see an entry here but no changes in any of these files : 

- `nix/deps/gradle/deps.json`
- `nix/deps/gradle/deps.urls`
- `nix/deps/gradle/deps.list`

did you run `make nix-update-gradle` ?

Reply:
nope, added it manually :| will fix.

Reply:
Okay cool, beware of this issue though : https://github.com/status-im/status-mobile/issues/18294
when you encounter this make sure you revert any changes in those 3 files i mentioned and in `nix/deps/gradle/proj.list` and run `make nix-update-gradle` again. repeat till you don't get any errors :)

good luck.

Reply:
done 👍 thanks for the help on this